Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/ThemeInspector.cs

```diff
@@ -151,7 +149,7 @@ protected bool RenderStates()
 
             if (drawerStarted)
             {
-                InspectorUIUtility.DrawSectionEnd(indentOnSectionStart);
+                InspectorUIUtility.DrawSectionEnd();
```

Comment:
DrawSection() no longer exists in InspectorUIUtility but is used in ThemeInspector.

Reply:
If you remove this whole condition, then the variable "drawerStarted" should also be removed anywhere it's used.

Reply:
Getting this cleared up got everything rendering properly and it looks really good now.

![image](https://user-images.githubusercontent.com/22085553/61488218-81b94980-a95c-11e9-9d56-28a583c4be08.png)

Reply:
Yes sorry, right as I was referencing you guys I noticed the bug and tried to delete my comment but not fast enough :P